tariff/octopus_test.go

```diff
@@ -31,5 +31,4 @@ func TestOctopusConfigParse(t *testing.T) {
 	}
 	_, err = NewOctopusFromConfig(invalidApiAndProductCodeConfig)
 	require.Error(t, err)
-
```

Comment:
strange linting issue here? my golangci-lint keeps automatically re-adding the whitespace to pass, had to manually yeet it with a hex editor